frontend/src/components/pages/InviteUsers.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import { Button, Input, Box, Text, Badge, Select } from "@chakra-ui/react";
+
+import NavigationBar from "../common/NavigationBar";
+import commonApiClient from "../../APIClients/CommonAPIClient";
+
+const InviteUsers = (): React.ReactElement => {
+  const [email, setEmail] = useState<string>("");
+  const [invitedEmail, setInvitedEmail] = useState<string>("");
+  const [invitedFirstName, setInvitedFirstName] = useState<string>("");
+  const [invitedLastName, setInvitedLastName] = useState<string>("");
+
+  const [userRole, setUserRole] = useState<string>("");
+  const [userStatus, setUserStatus] = useState<string>("");
+
+  const inviteUser = async () => {
+    await commonApiClient.inviteUser(
+      email,
+      userRole,
+      invitedFirstName,
+      invitedLastName,
+    );
+  };
+  const checkInvitedUser = async () => {
+    const res = await commonApiClient.isUserInvited(invitedEmail);
+    setUserStatus(res);
+  };
+  return (
+    <div className="page-container">
+      <NavigationBar />
+      <Box padding="20px">
+        <Text>Invite User:</Text>
+        <Input
+          type="text"
+          width="50%"
+          value={invitedFirstName}
+          onChange={(event) => setInvitedFirstName(event.target.value)}
+          placeholder="Enter first name"
+        />
+        <Input
+          type="text"
+          width="50%"
+          value={invitedLastName}
+          onChange={(event) => setInvitedLastName(event.target.value)}
+          placeholder="Enter last name"
+        />
+        <Select
+          placeholder="Enter role"
+          value={userRole}
+          onChange={(e) => setUserRole(e.target.value)}
+        >
+          <option value="Relief Staff">Relief Staff</option>
+          <option value="Admin">Admin</option>
+          <option value="Regular Staff">Regular Staff</option>
```

Comment:
I think it's better to throw these into an array and then simply map through the values ; will ensure that there's less copy-pasting if we ever want to extend this list.

```suggestion
         // Somewhere above in this file / in another file
         const ROLE_OPTIONS = ['Relief Staff', 'Admin', 'Regular Staff']

         // Specifically for lines 52-54
         {
               ROLE_OPTIONS.map((role) => <option value={role}>{role}</option>)
         }
```

Reply:
we won't actually be using this code for the feature, the front-end will be redone once we have the designs. Though having a list and mapping through it makes more sense than an option component for each item 👍 so we can make note of that later